config/muxcable.py

```diff
@@ -73,43 +73,48 @@ def lookup_statedb_and_update_configdb(per_npu_statedb, config_db, port, state_c
     ipv6_value = get_value_for_key_in_config_tbl(config_db, port, "server_ipv6", "MUX_CABLE")
 
     state = get_value_for_key_in_dict(muxcable_statedb_dict, port, "state", "MUX_CABLE_TABLE")
-    if (state == "active" and configdb_state == "active") or (state == "standby" and configdb_state == "active") or (state == "unknown" and configdb_state == "active"):
-        if state_cfg_val == "active":
-            # status is already active, so right back error
+    if state in ["active", "standby", "unknown"] and configdb_state in ["active", "manual"]:
```

Comment:
can we simplify line 76 to line 180 as follows?

```
if state_cfg_val is not configdb_state:
    config_db.set_entry("MUX_CABLE", port, {"state": state_cfg_val})
    port_status_dict[port] = 'INPROGRESS' if state_cfg_val is 'active' and state is not 'active' else 'OK'
else:
     port_status_dict[port] = 'OK'
```

Reply:
i think so, The logic is much more easier to follow. Thanks updated